documentation/high-level-introduction.md

```diff
@@ -0,0 +1,257 @@
+# A brief introduction to msbuild (Level 101)
+
+## Introduction
+
+This document aims to be a “101 college” level introduction into msbuild, useful for those coming to msbuild with very little background in how it works.
+
+### What is msbuild?
+
+msbuild is a build system commonly used to build .NET projects. It is made up the msbuild toolchain itself, and some number of additional files provided by the SDKs which provide platform specific instructions on how to process projects of those types.
```

Comment:
"It is made up the msbuild toolchain itself"
should be
"It is made up of the msbuild toolchain itself"?